serenity/ema_map.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef SERENITY_EMA_MAP_HPP
+#define SERENITY_EMA_MAP_HPP
+
+#include <string>
+
+#include <unordered_map>
+
+#include "filters/ema.hpp"
+
+#include "mesos/mesos.hpp"
+
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * Hasher functor for EmaMap
+ */
+struct EmaMapHasher{
```

Comment:
You should name it ExecutorInfoHasher and this piece of code should be reusable in other components.